### ISSUE TITLE
Bootstrapping instructions are broken

### ISSUE THREAD

**Issue by atsampson** [ocaml/dune#995]
When bootstrapping without opam, README says to do:

```
$ ocaml bootstrap.ml
$ ./boot.exe
$ ./_build/default/bin/main.exe install
```

This needs fixing for the dune/jbuilder split:

```
$ ocaml bootstrap.ml
$ ./boot.exe
$ ./_build/default/bin/main_dune.exe build @install
$ ./_build/default/bin/main_dune.exe install
```

**Comment by diml** [comment]
Indeed, thanks for the report. The `main_dune.exe build @install` line is not needed BTW, that's what `./boot.exe` does.

**Comment by atsampson** [comment]
If I don't do `build @install`, then the `install` step fails for me:

```
$ ocaml bootstrap.ml
[... snip output ...]
$ ./boot.exe
$ ./_build/default/bin/main_dune.exe install
The following <package>.install are missing:
- _build/default/jbuilder.install
You need to run: dune build @install
```

**Comment by diml** [comment]
Ah, that should be fixed by #994

**Comment by diml** [comment]
The other solution is to do `./_build/default/bin/main_dune.exe install dune`
